src/main.tsx

```diff
@@ -1,5 +1,5 @@
 import { StrictMode } from 'react'
-import ReactDOM from 'react-dom/client'
```

Comment:
`scrivitoExtensions.tsx` uses `import * as ReactDOM from 'react-dom/client'`. Could you switch this to direct import as well?